pkg/provisioningv2/rke2/planner/planner.go

```diff
@@ -693,6 +693,26 @@ func addRoleConfig(config map[string]interface{}, controlPlane *rkev1.RKEControl
 		config["disable-etcd"] = true
 	}
 
+	if isControlPlane(machine) {
+		certArg := fmt.Sprintf("cert-dir=/var/lib/rancher/%s/server/tls/kube-controller-manager", rancherruntime.GetRuntime(controlPlane.Spec.KubernetesVersion))
+		securePortArg := "secure-port=10257"
+		if v, ok := config["kube-controller-manager-arg"]; ok {
+			switch v.(type) {
+			case []string:
+				config["kube-controller-manager-arg"] = append(v.([]string), certArg, securePortArg)
```

Comment:
Do we need to check to see if the user has already given these settings in this array?